jmix-flowui/flowui-themes/src/main/resources/META-INF/resources/themes/jmix-lumo/component/jmix-upload-field.css

```diff
@@ -26,6 +26,7 @@ jmix-upload-field[theme~="no-file-name"] .jmix-upload-button {
 
 .jmix-upload-field-file-name {
     cursor: pointer;
+    margin: 0;
```

Comment:
Not relevant to the current issue. Create a new one with motivation

Reply:
https://github.com/jmix-framework/jmix/issues/2299